bot-components/GitHub_mutations.ml

```diff
@@ -201,9 +201,9 @@ let send_status_check ~bot_info ~repo_full_name ~commit ~state ~url ~context
     repo_full_name commit state
   >>= fun () ->
   let body =
-    "{\"state\": \"" ^ state ^ "\",\"target_url\":\"" ^ url
-    ^ "\", \"description\": \"" ^ description ^ "\", \"context\": \"" ^ context
-    ^ "\"}"
+    {|{"state": "|} ^ state ^ {|","target_url":"|} ^ url
+    ^ {|", "description": "|} ^ description ^ {|", "context": "|} ^ context
+    ^ {|"}|}
```

Comment:
This begs the question: isn't there a good antiquotation mechanism in OCaml that would make this string even more readable (we could always rely on `Printf.fprintf`, which I have abbreviated into `f` in this code base, of course, but it's not terribly satisfying).

Reply:
I wanted to talk about that too :D 
The thing is, if we use `sprintf` and write the arguments in the wrong order, the bugs will be really hard to fine so the current solution, even though hard to read, is the best currently in my opinion.

I'm waiting for named parameters in formatting strings that would allow us to write:

`{|{"state": "{state}", "target_url":"{url}", "description": "{description}", "context": "{context}"}|}`

But in the meantime, I'd stick to this